roles/deploy/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Invoke deploy role
+  ansible.builtin.include_role:
+    name: network.base.deploy
```

Comment:
network.base not required here, we just need to convert existing network.vpn 
deploy operation into the deploy role